data-and-functions-1/src/functions/getProductById.js

```diff
@@ -0,0 +1,8 @@
+const getProductById = (data, id) => {
+  if (data == null || data.products == null || id == null) {
+    return null
+  } else {
+    return data.products.find((p) => p.id === id)
```

Comment:
When your lambda function only takes one parameter, you can eliminate the parens if you want. I find it makes the code easier to read:
```
p => p.id === id
```
Your call on that one though.